### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by hne3** [hne3/EECS-393-Nutrition-App#3]
Need a tutorial to aid new users in learning site functionality

**Comment by oeihiko** [comment]
Is this supposed to be interactive?

**Comment by hne3** [comment]
Yes, that would be preferable.

**Comment by oeihiko** [comment]
Was an interactive tutorial specified in any of our design documentation?

**Comment by 12wrigja** [comment]
Nope.

**Comment by oeihiko** [comment]
Then do we really want to be doing it? Interactive tutorials aren't trivial, afaik.

**Comment by 12wrigja** [comment]
^^ They aren't. Believe me, anything interactive is a LOT of javascript...

**Comment by hne3** [comment]
This was specified in our SRS, actually. Doesn't have to be interactive if it's going to be too involved, but we did say we'd do it.

**Comment by 12wrigja** [comment]
"A basic tutorial will exist on the website to help jumpstart use of the less-common product features (for release 3.0.)"

So, not interactive by my standards.

**Comment by hne3** [comment]
I agree that it shouldn't be interactive if it's going to be too much effort. This task is intended to be a quick way to get users on-board with Snackr, and isn't critical right now, but should be done.

**Comment by lucyktan** [comment]
Matt, your only other issue right now besides the tutorial is the FAQ page, which shouldn't take much effort or time to set up. I think it'd be fair for you to either work on an interactive tutorial or be assigned additional issues.

**Comment by oeihiko** [comment]
Right, @lucyktan, that's my point. If we decide that an interactive tutorial is not necessary for our project, then I can be assigned additional issues. Right now I'm working on the FAQ, which by nature will also help new users. We can look at it after I'm done and then decide how much more tutelage we need.

**Comment by lucyktan** [comment]
Okay, that's great! It'd be really nice if you could help with some of the main functionality, so feel free to assign yourself to existing issues or create new issues to work on. Thanks!
